cdn/package.json

```diff
@@ -1,5 +1,5 @@
 {
-	"name": "@fosscord/cdn",
+	"name": "cdn",
```

Comment:
I would like to keep @fosscord as I'll use it in the server bundle and we maybe even could upload it to npm